packages/bigquery/test/test.js

```diff
@@ -138,7 +148,11 @@ test('numeric types are retrieved correctly', async () => {
 test('query batches results properly', async () => {
 	const { rows, expectedRowCount } = await runQuery(
 		'select 1 union all select 2 union all select 3 union all select 4 union all select 5',
-		undefined,
+		{
+			project_id: process.env.BIGQUERY_PROJECT_ID,
+			client_email: process.env.BIGQUERY_CLIENT_EMAIL,
+			private_key: process.env.BIGQUERY_PRIVATE_KEY
```

Comment:
Probably worth putting this object and the other 2 into a `const creds = { ... }`